hello_world.py

```diff
@@ -0,0 +1,2 @@
+#This is a file to test the github exercise and have a python file.
+print("Hello World")
```

Comment:
nice! 

We will however not merge this in as it has nothing to do with the detector. 

This comment will show you how to do a code review, as we should for our students.

Reply:
Cool!

Reply:
I should close the pull request at this point then?